modular_skyrat/modules/modular_vending/code/cigarette.dm

```diff
@@ -11,3 +11,6 @@
 		/obj/item/reagent_containers/vapecart/pwrgame = 4,
 		/obj/item/reagent_containers/vapecart/cheese = 4,
 	)
+	skyrat_premium = list(
```

Comment:
these should DEFINITELY be contraband, not premium, based on their chemical makeup.

Reply:
> these should DEFINITELY be contraband, not premium, based on their chemical makeup.

I see your point, but they're also a donator item that I wanted to avoid putting in the donator's loadout while still making them available without too much trouble.
Might tone down the chem contents though.

Reply:
seconding this